src/main/scala/zio/prelude/package.scala

```diff
@@ -103,9 +104,10 @@ package object prelude
     type Applicative[F[+_]]         = Covariant[F] with IdentityBoth[F]
     type InvariantApplicative[F[_]] = Invariant[F] with IdentityBoth[F]
 
-    type Category[:=>[-_, +_]]   = IdentityCompose[:=>]
-    type Profunctor[:=>[-_, +_]] = Divariant[:=>]
-    type Bifunctor[:=>[+_, +_]]  = Bicovariant[:=>]
+    type Category[:=>[-_, +_]]      = IdentityCompose[:=>]
+    type Profunctor[:=>[-_, +_]]    = Divariant[:=>]
+    type Bifunctor[:=>[+_, +_]]     = Bicovariant[:=>]
+    type Zifunctor[:=>[-_, +_, +_]] = Zivariant[:=>]
```

Comment:
Yes, it would be good to remove it, there is not traditional name `Zifunctor`.
These are aliases defined in a `classic` object after all :wink:

Reply:
Done